clarity_ext/context.py

```diff
@@ -196,6 +196,104 @@ def commit(self):
             obj.commit()
 
 
+class FakingEntityMonkey:
```

Comment:
What I'm thinking is that "monkey patching" is similar to "overriding". I would be confused if I saw a class called `EntityOverrider`. I think the technology used to achieve the result is not the main thing. But I would like to show you if I may how I would approach this, in a different branch.